.github/workflows/deploy.yml

```
@@ -0,0 +1,61 @@
+    runs-on: ubuntu-latest
+    environment: prd
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4.0.2
+        with:
+          role-to-assume: arn:aws:iam::${{ vars.AWS_ACCOUNTID }}:role/GithubTrigger
+          aws-region: ${{ env.AWS_REGION }}
+      - name: Deploy Code
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:4:61: Double quote to prevent globbing and word splitting

(shellcheck)